code/modules/reagents/reagent_containers/cups/_cup.dm

```diff
@@ -195,6 +210,40 @@
 	possible_transfer_amounts = list(5,10,15,20,25,30,50,100)
 	label_icon = "label_beakerlarge"
 
+/obj/item/reagent_containers/cup/beaker/vial
+	name = "Vial"
+	desc = "A  very small recipent for the strongest potions for going into battle. That is, if you can handle them. Can hold up to 100 units." //https://www.youtube.com/watch?v=R_FQU4KzN7A
```

Comment:
I don't think the description should be quite this whimsical, but more importantly this vial does not hold 100u and the amount the vial holds shouldn't be part of the description like this even if it did. `examine()` already pulls this information dynamically when appropriate.